Updating minimum required versions of Python and selenium.

aaltat opened the pull request:
Also contains documentation updates.

aaltat committed:
Updating minimum required versions of Python and selenium.

Also contains documentation updates.

aaltat committed:
More document refactoring

aaltat commented:
There is `easy_install` in the documentation, do we need to keep that one too in the docs?

pekkaklarck commented:
Definitely no need to mention easy_install. Don't think proxy stuff is that important here either. Could link to Robot's more comprehensive docs about that and other more advanced pip/installation topics. Or to pip's own docs.

aaltat commented:
OK, I will delete the `easy_install` chapter. Then there is only `pip` and source install options.

Linking to other documents is also good idea. It would be nice to know is the Jython supported in S2L, but not in this release.

aaltat committed:
Updated INSTALL.rst

Removed eays_install sections. Added chapter about selenium
need for browser driver and link to the correct selenium
documentation. Also added chapter about pip support for direct
installation from GitHub repository.
